### DISCUSSION TITLE
Add document history to search results

### DISCUSSION THREAD

**Discussion by msklywenn** [outline/outline#6540] [Ideas]
It would be nice if search could (optionally) return results found in older revisions of documents.

**Comment by tommoor** [comment]
Mmmm, good request! We don't maintain a search index for the many revisions that can exist in the past but it could be that a simpler matching engine is used against the text content directly. 

Did you have a usecase recently that needed this functionality?

**Comment by msklywenn** [comment]
Yes, I needed to find something I thought we had written. I wasn't sure if it was in our chat or in our outline. So I tried finding in Outline but nothing came up. Then I tried to check if it could find anything that was removed from a document and noticed it didn't. So I had to manually search through the history of some pages and eventually found something.
